app/assets/stylesheets/recipe_new.scss

```diff
@@ -139,3 +139,7 @@
 .new-recipe-wrapper {
   margin-left: 50px;
 }
+
+input#recipe_shared {
```

Comment:
Avoid using id selectors<br>Avoid qualifying id selectors with an element.<br>Selector `recipe_shared` should be written in lowercase with hyphens